docs/notebooks/local_frequency_analysis.ipynb

```diff
@@ -9,9 +9,138 @@
     },
```

Comment:
<p>Je propose ceci. Aussi, pas besoin de répéter la dernière phrase, je crois:</p><p> </p><p><code>xhfa.local._get_plotting_positions</code> allows you to get plotting positions for all variables in the dataset. &nbsp;It accepts <code>alpha</code>  <code>beta</code>  arguments. See the <a href="https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.mstats.plotting_positions.html" target="_blank">SciPy documentation</a> for typical values. By default, (0.4, 0.4) will be used, which corresponds to <code>approximately quantile unbiased (Cunnane)</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/hydrologie/xhydro/pull/87/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>